media/videostitcher/stitcher_test.go

```diff
@@ -28,10 +28,20 @@ import (
 )
 
 const (
-	deleteCdnKeyResponse = "Deleted CDN key"
-	deleteSlateResponse  = "Deleted slate"
-	location             = "us-central1" // All samples use this location
-	slateID              = "my-go-test-slate"
+	location            = "us-central1" // All samples use this location
+	slateID             = "my-go-test-slate"
+	deleteSlateResponse = "Deleted slate"
+
+	deleteCdnKeyResponse  = "Deleted CDN key"
+	gcdnCdnKeyID          = "my-go-test-google-cdn"
+	akamaiCdnKeyID        = "my-go-test-akamai-cdn"
+	hostname              = "cdn.example.com"
+	updatedHostname       = "updated.example.com"
+	gcdnKeyname           = "gcdn-key"
+	gcdnPrivateKey        = "VGhpcyBpcyBhIHRlc3Qgc3RyaW5nLg=="
+	updatedGcdnPrivateKey = "VGhpcyBpcyBhbiB1cGRhdGVkIHRlc3Qgc3RyaW5nLg=="
+	akamaiTokenKey        = gcdnPrivateKey
```

Comment:
This and the next car are not needed if there are just the same string as above.

Reply:
Removed and renamed existing vars